doc/crypto/BIO_new.pod

```diff
@@ -38,6 +39,8 @@ BIO_new() returns a newly created BIO or NULL if the call fails.
 
 BIO_set(), BIO_free() return 1 for success and 0 for failure.
 
+BIO_up_ref() increments the reference count associated with the BIO object.
```

Comment:
This should be in the DESCRIPTION section, with BIO_up_ref() additionally added to the sentence above with the list of functions returning 1 for success and 0 for failure.